bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -43,8 +43,9 @@ class Tooltip extends Component {
     } = this.props;
 
     const options = {
-      position,
-      dynamicTitle: this.dynamicTitle,
+      placement: position,
+      performance: true,
+      // dynamicTitle: this.dynamicTitle,
```

Comment:
as far I understand this option is not needed/exists anymore in the new version.